### ISSUE TITLE
Train 0.7.0  acoustic model for English

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#2648]

**Comment by dabinat** [comment]
Would it be possible to train with mixed precision for 0.7? Is there a reason it wasn't activated for 0.6?

**Comment by reuben** [comment]
I guess for now we'd rather pay the price of somewhat longer training runs than have to figure out yet another hyperparameter. Do you get similar accuracy with faster training on your tests?

**Comment by dabinat** [comment]
@reuben Well that's the problem: I'd like to use it but I'm training from the pretrained checkpoint so I'm limited to the parameters that were originally used. But if you need feedback, I can do some tests on a new training run and see what I come up with.

**Comment by rhamnett** [comment]
I vote for this as there is almost no difference in the final accuracy / WERs

**Comment by PedroDKE** [comment]
Is it already known if the 0.7.0 model will work with deepspeech 0.6.1 and other way around?

**Comment by reuben** [comment]
The 0.7.0 checkpoint will NOT work with the DeepSpeech 0.6.1 training graph, and the 0.7.0 model will NOT work with the DeepSpeech 0.6.1 clients.

**Comment by kdavis-mozilla** [comment]
Finished with job 5562, lm_alpha=0.931289039105002, lm_beta=1.1834137581510284

**Comment by liziru** [comment]
@reuben where can i find 0.7.0 pre-trained model? I have searched for 0.7.0 pre-trained model for a long time. I will appreciate it for any help.

**Comment by kdavis-mozilla** [comment]
@liziru It will be made public when the 0.7.0 release is made public.

**Comment by tom-melistas** [comment]
@kdavis-mozilla any idea when will that be? or if there is any way to make a pretrained 0.6 model compatible with 0.7

**Comment by kdavis-mozilla** [comment]
@tom-melistas In the next 2-3 weeks.
